examples/agent_motion_prediction/agent_motion_config.yaml

```
@@ -7,12 +7,8 @@ model_params:
   model_architecture: "resnet50"
 
   history_num_frames: 0
-  history_step_size: 1
-  history_delta_time: 0.1
-
   future_num_frames: 50
-  future_step_size: 1
-  future_delta_time: 0.1
+  delta_time: 0.1
```

```suggestion
  step_time: 0.1
```
How about call it `step_time`? Be consistent with the variable name used in the code.